src/ert/_c_wrappers/job_queue/environment_varlist.py

```diff
@@ -16,18 +16,29 @@ class EnvironmentVarlist(BaseCClass):
         "void env_varlist_update_path(env_varlist, char*, char*)"
     )
 
-    def __init__(self, _vars: Optional[Dict[str, str]] = None):
-        if _vars is None:
-            _vars = {}
+    def __init__(
+        self,
+        vars: Optional[Dict[str, str]] = None,
+        paths: Optional[Dict[str, str]] = None,
+    ):
+        if vars is None:
+            vars = {}
+        if paths is None:
+            paths = {}
         c_ptr = self._alloc()
         super().__init__(c_ptr)
 
-        for key, value in _vars.items():
+        for key, value in vars.items():
```

Comment:
nitpick; pylint complain
`Redefining built-in 'vars' (redefined-builtin)`

Reply:
Ah, so that is why that was renamed, `env_vars` then?